concepts/chars/about.md

```diff
@@ -0,0 +1,141 @@
+Also, lots of new bugs were introduced.
+
+To prevent _everything_ breaking, the Unicode/UTF-8 design ensures that the first 127 codes are identical to ASCII _(even the bell)_.
+
+## Characters in Kotlin
+
+Languages designed after about 2005 have the huge advantage that a reasonably stable Unicode standard already existed.
+
+Kotlin (first released in 2011) was able to assume that users would use a variety of (human) languages, and would need Unicode to express them.
+
+~~~~exercism/advanced
+[Characters][ref-char] in Kotlin are 16-bit (UTF-16) [`codepoints`][wiki-codepoint], the same as a JVM `char`.
+This is enough to express most written alphabets, but not the entire range of emojis.
+
+The full Unicode standard uses up to six bytes (48 bits) per character (called a [`grapheme`][wiki-grapheme]).
+
+Kotlin `Strings` support this full standard by using multiple codepoints per character, when necessary.
+For example, 😱 would be `\uD83D` and `\uDE31`.
+
+Unfortunately, Java has no built-in grapheme support, and for compatibility neither does Kotlin.
+
+[wiki-codepoint]: https://en.wikipedia.org/wiki/Code_point
+[wiki-grapheme]: https://en.wikipedia.org/wiki/Grapheme
+[ref-char]: https://kotlinlang.org/docs/characters.html
+~~~~
+
+
+Character literals are written in single-quotes, and are distinct from strings written in double quotes.
+This is probably obvious to people from the C/C++ world, but potentially confusing to Python and JavaScript programmers.
+
+```kotlin
+val a = 'a'
+a::class.qualifiedName  // => kotlin.Char
+a.code  // => 97
+
+val jha = 'झ'  // Devanagari alphabet
+jha.code  // => 2333
+
+val heart = '❤'  // heart emoji
+heart.code  // => 10084
+
+Char.MAX_VALUE.code  // => 65535 (64k, the largest code point allowed)
+```
+
+Converting between `Char` and `Int` is simple:
+
+```kotlin
+a.code  // => 97  (a.toInt() is deprecated)
+Char(97) // => 'a'
+```
+
+The compiler allows _some_ forms of integer arithmetic on `Char`s:
+
+```kotlin
+'a' + 5       // => 'f'
+'c' - 'a'   // => 2
+'c' + 'a'   // => error!
+
+'f' + ('A' - 'a')  // => 'F' (same as 'f'.uppercase()
+
+'f'.dec() // => 'e' (decrement)
+'f'.inc() // => 'g' (increment)
+```
+
+## Some functions for `Char`
+
+As always, there are far too [many functions][ref-char-lib] to discuss here, so this is just a selection.
+
+- For appropriate alphabets, change case with [`uppercase()`][ref-uppercase] and [`lowercase()`][ref-lowercase].
+- Test case with [`isUpperCase()`][ref-isuppercase] and [`isLowerCase()`][ref-islowercase].
+- Test character type with:
+    - [`isLetter()`][ref-isletter], covers many alphabets
+    - [`isDigit()`][ref-isdigit], in range 0..9
+    - `isLetterOrDigit()`, combines the previous two
+    - [`isWhitespace()`][ref-iswhitespace], any whitespace character
```

Comment:
```suggestion
- Test character type with:
  - [`isLetter()`][ref-isletter], covers many alphabets (the Lu, Ll, Lt, Lm, and Lo categories in unicode)
  - [`isDigit()`][ref-isdigit], in range 0..9 (the Nd category in unicode)
  - `isLetterOrDigit()`, combines the previous two
  - [`isWhitespace()`][ref-iswhitespace], any whitespace character (the Cc, Zp, Zl, and Zs categories in unicode)
```

I think this is the correct markdown format. 

For about.md only, I think we can explain how it's implemented by specifying the unicode categories (that's how these functions work!)